nano/node/active_transactions.cpp

```diff
@@ -582,7 +582,7 @@ nano::vote_code nano::active_transactions::vote (std::shared_ptr<nano::vote> vot
 	{
 		if (processed)
```

Comment:
This is supposed to include && !is_half_pr()

Reply:
Assuming you meant `!node.wallets.rep_counts ().have_half_rep ()` this is now handled